File: src/main/kotlin/com/randomperson22/piggymodtest/entities/sheepy/SheepyRenderer.java
package com.randomperson22.piggymodtest.entities.sheepy;

import net.minecraft.client.renderer.entity.RenderManager;
import software.bernie.geckolib3.renderers.geo.GeoEntityRenderer;

public class SheepyRenderer extends GeoEntityRenderer<EntitySheepy> {
    public SheepyRenderer(RenderManager renderManager) {
        super(renderManager, new SheepyModel());
        this.shadowSize = 0.5F;
    }
}

File: src/main/kotlin/com/randomperson22/piggymodtest/entities/sheepy/SheepyRenderer.kt
package com.randomperson22.piggymodtest.entities.sheepy

import net.minecraft.client.renderer.entity.RenderManager
import software.bernie.geckolib3.renderers.geo.GeoEntityRenderer

class SheepyRenderer(renderManager: RenderManager?) : GeoEntityRenderer<EntitySheepy?>(renderManager, SheepyModel()) {
    init {
        this.shadowSize = 0.5f
    }
}